VAMobile/e2e/tests/ContactInformation.e2e.ts

```diff
@@ -402,6 +402,7 @@ export async function verifyNonUSorMilitaryAddresses(addressID: string, addressT
   })
 }
 beforeAll(async () => {
+  await toggleRemoteConfigFlag(CommonE2eIdConstants.IN_APP_REVIEW_TOGGLE_TEXT)
```

Comment:
Can we toggle this globally for all E2E tests instead of toggling for individual sets of tests? And then remove all the calls to resetInAppReview in all E2E tests? It would really speed up the tests I think